source/SpinalHDL/Data types/bits.rst

```diff
@@ -202,7 +201,7 @@ Type cast
      -  Bits(w(x) bits)
 
 
-To cast a Bool, UInt or a SInt into a Bits, you can use ``B(something)``
+To cast a ``Bool``, ``UInt`` or an ``SInt`` into a ``Bits``, you can use ``B(something)`` as in the example below:
```

Comment:
Mmm, I think that just use ":" is the way to go here. What do you think?
```suggestion
To cast a ``Bool``, ``UInt`` or an ``SInt`` into a ``Bits``, you can use ``B(something)``:
```

Reply:
I agree, and will truncate it down to just the colon.